deploy-manage/deploy/cloud-on-k8s/install-using-yaml-manifest-quickstart.md

```diff
@@ -14,5 +17,79 @@ mapped_urls:
 
 % Use migrated content from existing pages that map to this page:
 
+% removed both
 % - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-install-yaml-manifests.md
-% - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-deploy-eck.md
\ No newline at end of file
+% - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-deploy-eck.md
+
+This method is the quickest way to get started with ECK if you have full administrative access to the Kubernetes cluster. 
+
+During the installation, the following components are installed or updated:
+
+* `CustomResourceDefinition` objects for all supported resource types (Elasticsearch, Kibana, APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash).
+* `Namespace` named `elastic-system` to hold all operator resources.
+* `ServiceAccount`, `ClusterRole` and `ClusterRoleBinding` to allow the operator to manage resources throughout the cluster.
+* `ValidatingWebhookConfiguration` to validate Elastic custom resources on admission.
+* `StatefulSet`, `ConfigMap`, `Secret` and `Service` in `elastic-system` namespace to run the operator application.
+
+## Prerequisites and considerations
+
+Before you begin, review the following prerequisites and recommendations:
+
+* For this quickstart guide, your Kubernetes cluster is assumed to be already up and running. Before you proceed with the ECK installation, make sure you check the [supported versions](/deploy-manage/deploy/cloud-on-k8s.md#k8s-supported).
+
+* If you are using GKE, make sure your user has `cluster-admin` permissions. For more information, check [Prerequisites for using Kubernetes RBAC on GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#iam-rolebinding-bootstrap).
+
+* If you are using Amazon EKS, make sure the Kubernetes control plane is allowed to communicate with the Kubernetes nodes on port 443. This is required for communication with the Validating Webhook. For more information, check [Recommended inbound traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.md).
+
+* Refer to [*Install ECK*](../../../deploy-manage/deploy/cloud-on-k8s/install.md) for more information on installation options.
+
+* Check the [upgrade notes](../../../deploy-manage/upgrade/orchestrator/upgrade-cloud-on-k8s.md) if you are attempting to upgrade an existing ECK deployment.
+
+##  Installation procedure
+
+To deploy the ECK operator:
+
+1. Install [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) with [`create`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/):
+
+    ```sh
+    kubectl create -f https://download.elastic.co/downloads/eck/{{eck_version}}/crds.yaml
+    ```
```

Comment:
this variable `{{eck_version}}` does not exist in `docset.yml` and needs to be added.

not sure if it will display after it's added - if it doesn't, we'll need to log it as a bug. according to martijn [it should work fine](https://elastic.slack.com/archives/C05PP2LEC1X/p1739387960361149)

Reply:
making note of it in the parent issue. left for the next PR.